app/src/main/java/fr/voxeet/sdk/sample/application/SampleApplication.java

```diff
@@ -32,8 +33,8 @@ public void onCreate() {
         UXKitLogger.logcatEnabled = true;
 
         VoxeetSDK.initialize(
-                "iYvHTzKMKBTPAK1j1RmTYA==",
-                "aD9RhnxDc34hp9ns88X95gSjFZt6mYuD1XitDx9zB-I="
+                BuildConfig.CONSUMER_KEY,
```

Comment:
you will need to force push a squashed version to remove it entirely from the history